_config.yml

```diff
@@ -2,7 +2,7 @@ plugins:
   - jekyll-paginate-v2
   - jekyll-polyglot
   - jekyll-minifier
-languages:                   ["en", "ko_KR", "pt_BR", "nl", "es", "de", "it", "fr", "pt"]
+languages:                   ["en", "zh_CN", "nb_NO", "ko_KR", "pt_BR", "nl", "es", "de", "it", "fr", "pt"]
```

Comment:
We need to find a better way on how to show these many languages. It's slowly getting full on the sidebar ;-)